src/psyclone/psyGen.py

```diff
@@ -7001,7 +7006,8 @@ def gen_c_code(self, indent=0):
         :param int indent: Depth of indent for the output string.
         :return: C language code representing the node.
         :rtype: str
-        :raises GenerationError: if the node or its children are invalid.
+        :raises GenerationError: If the node or its children are invalid.
```

Comment:
(Picky) but following our discussion the other day, please could you put blank lines between the sections documenting the arguments, return value and exceptions.

Reply:
Done